DESCRIPTION

```diff
@@ -65,7 +65,13 @@ Imports:
     xgboost
 Remotes:
     mlr-org/mlr3book,
+<<<<<<< HEAD
+    mlr-org/mlr3fselect,
+    mlr-org/mlr3pipelines,
+    mlr3learners/mlr3learners.randomforest
+=======
     mlr-org/mlr3extralearners
+>>>>>>> master
```

Comment:
@be-marc FYI

Reply:
I should not push and pack in. Thanks :)